src/app/voting/new-voting-button/new-voting-button.component.html

```diff
@@ -0,0 +1 @@
+<button (click)="onVtnClick()" mat-raised-button ><mat-icon>add</mat-icon> Neues Voting </button>
```

Comment:
```suggestion
<button (click)="onVtnClick()" class="btn btn-primary" mat-raised-button ><mat-icon>add</mat-icon> Neues Voting </button>
```
Is it necessary to put this button in a separate component?

Reply:
@Gerw4ld